indico/htdocs/sass/partials/_colorswitch.scss

```diff
@@ -0,0 +1,72 @@
+/* This file is part of Indico.
+ * Copyright (C) 2002 - 2015 European Organization for Nuclear Research (CERN).
+ *
+ * Indico is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation; either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * Indico is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Indico; if not, see <http://www.gnu.org/licenses/>.
+ */
+
+.color-palette {
+    background: #FFF;
+    background-clip: padding-box;
+    border: thin solid lighten($gray, 10%);
```

Comment:
Does it look very ugly with `$color-variation` instead of arbitrary percentage? The idea of palette+`$color-variation` is that we don't end up with arbitrary shades of colors, but we stick to color +- a predefined variation. I know it's not always possible to make it look nice, but let's stick to it if possible.